tests/Issue2030.py

```diff
@@ -0,0 +1,18 @@
+from . import Framework
+
+
+class Organization(Framework.TestCase):
+    def testIssue2030CreateProject(self):
+        super().setUp()
+        project = "ultratendency"
+        self.user = self.g.get_user("karthik-kadajji-t")
+        self.org = self.g.get_organization("testkarthik")
+        self.org.create_project(project)
+        success = False
+        projects = self.org.get_projects()
```

Comment:
This is an expensive operation, can't you look up the project by name?

Reply:
No, We cannot get the project by the name. Since get_projects returns PaginatedList . But I could create a organization with no projects And then run the modified tests to fetch just the 1st element in the list .
project = self.org.get_projects()[0].name
This would assure that we could end the iteration early. Also since this is just for testing, I believe it wouldn't affect the overall performance of the library.
I will modify with the above changes and give a PR:)

Reply:
@s-t-e-v-e-n-k Would you think any more changes is need on the new PR?